tasks/TOPSIM - general mgmt business game.md

```diff
@@ -0,0 +1,47 @@
+2. Following the random assignment of students to teams they are no longer allowed to withdraw from the experiment, nor are they permitted to switch teams. Nevertheless, participants can simply stop attending sessions and team meetings. 
+
+3. Teams then internally agree upon where, how often and for how long they meet to debate their strategies. Thus, in some teams decisions may be worked out together in group meetings, while in others, team members specialize on different functional areas and discuss their suggestions at a weekly summit
+
+4. Over each period (the eight weeks), teams have to develop and adapt business strategies based on the basic endowments in the different functional areas of their company and based on the economic situation in the respective market. Both aspects are determined by the software at the beginning of each period and are identical for all teams in a particular market. Companies’ progress is a function of how teams as well as the competitors react to the market conditions. 
+> In each period, teams have to decide on a number of variables driving company performance: the sales price(s) of the product(s), marketing and sales activities, R&D expenses, investment in environmental facilities or process optimizations, and financial resources.
+> A manual helps students to develop appropriate strategies. Teams that are able to cope with these various challenges and to make rational decisions after carefully weighting all opportunities and risks have the highest success probability.
+
+5. At the end of each period, each group transmits its decision to a course administrator, who processes all the data and simulates each team’s business performance of that particular period. 
+> The applied software uses different algorithms to calculate each company’s periodic share price based on how the company as well as their competitors decided on the above-described variables given the predetermined endowment and economic circumstances. 
+> The group then receives a full report documenting how the team performed as well as an updated economic forecast for the subsequent period in order to make new business decisions.
+
+## Roles 
+A description of roles if relevant to the task.  
+
+If relevant, describe how roles are assigned and used in the task. List all roles and list how the experience for each role differs from others.
+
+## Instructions
+Instructions are included in TOPSIM Game: needs to be purchased: https://www.topsim.com/en/education-solutions/
+
+
+# Criteria
+## Performance calculation
+
+The team performance measures in our analyses are the firm’s final share price and whether a particular firm went bankrupt or not. Information on the share price was retrieved from firms’ reports after the last particular period had been simulated
+
+## Incentives
+No incentives as students played this game as a part of their course and performance impacted final grades. 
```

Comment:
Final grades were cited as the incentive, but I also agree with how you have it stated, whatever you think is better.